### PR TITLE
Add weekly status calls note to contributing doc

### LINKED ISSUE #256: Open weekly BitOps calls
we'd like to host a weekly call for BitOps so that the community can join.

Things to do:
- [ ] Create a recurring event
- [ ] Link to the public event details in the docs
- [ ] Notify community slack
- [ ] Include notice on BitOps page

### PR DESCRIPTION
**Pull request by mickmcgrath13**
# Description

We'd like to start hosting a public weekly call

Fixes #256 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ We'd love to accept your patches and contributions to this project. There are ju
 
 When you are ready to get started developing, see our [development guide](development/development.md) for how to get started!
 
+## Weekly status calls
+
+Join us weekly on Mondays at 10 AM Eastern time!
```
Comment (armab) [inline docs/contributing.md:8]: Great stuff!

Just a couple of pros/cons notes about the timing: 
- 10am Eastern is 7am Western which might be too early for that part.
- Mondays are hard for everyone and so people are busy with their own work.

Those might be limiting factors for someone joining the community calls and "fun" stuff.

But I think it's great to start it the way it's proposed and maybe optimizing in the future (if there's a need).

The advantage is that doing it on Mondays would feel like a BitOps week planning community call prioritizing the development that might be more important at this point 👍

**Comment by mickmcgrath13** [comment]
@armab What do you want to do with this ticket?  Should we update the docs in addition to https://github.com/bitovi/bitops/discussions/288 ?

**Comment by mickmcgrath13** [comment]
@armab What do you want to do with this ticket?  Should we update the docs in addition to https://github.com/bitovi/bitops/discussions/288 ?

**Comment by armab** [comment]
Good call. We'll definitely need to sync these docs 👍

**Review by armab — APPROVED** [review]

**Pull request merged**